### DISCUSSION TITLE
EasyRPG?

### DISCUSSION THREAD

**Discussion by arist0brat** [eduardofilo/RG350_adam_image#295] [Q&A] [answered]
Has anyone been able to setup EasyRPG on the Adam image? I know it's been done on other images, but I really like the way that this one is set up. 

I'm able to get the OPK itself working by putting it on the external card in the "apps" folder (I'm on a 280v), but I'm unable to get it to detect games no matter where I put them. I tried following the tutorial [here](https://retrogamecorps.com/2020/09/11/guide-rg350-native-apps-and-games/#RPGMaker) since the install process for most things are very similar between the 350 and the 280v but I think it's based on the folder structure of the default firmware.

**Comment by a-dekker** [comment, marked as answer]
No issues loading games here. Once you start the opk you will get a default file browser screen. I extracted the Dungeoneer rar file on the sdcard en selected Dungeoneer.exe from there to start the game itself.

If you have an RG350M, do not use the OPK from your link, but the one from the Compatible game ports [here](https://github.com/eduardofilo/RG350_adam_image/wiki/En:-8.-Compatible-game-ports). The first one does not scale for a higher resolution.

**Comment by a-dekker** [comment]
I get the impression you are starting EasyRPG via Simplemenu. In that case you will indeed get a "No games found in the current directory.". I am starting the EasyRPG using the default GMenu2x frontend.

**Edit**: did a little hack to make it work with Simplemenu:

The opk already creates the folder `/media/data/local/home/.easyrpg`. If you now place a file called "`project-path.txt`" in that folder and add the path to the folder with gamefolders in there (e.g. `/media/sdcard/EasyRPG/`), EasyRPG will show its own startmenu with all subfolders you can pick for game start.

One caveat, as long as the project_path.txt is not created the opk will act as the old one. If the file is found and has a valid  location you have a fixed game location. From within GMenu2x you still get the file browser, but you will always and up in the EasyRPG startmenu with your games from the fixed location. Which is what you probably want anyway. Removing the "project-path.txt" will end up in the old free browsing mode without the EasyRPG startmenu (and no longer working in Simplemenu).

Modified OPK: https://github.com/a-dekker/repackaged-opks/raw/main/EasyRPG_gcw0_rg350.opk

**Reply by eduardofilo** [reply]
It can also be used from SimpleMenu, but in this case the corresponding entry must be added to some of the groups, for example in the Home Computer. Something like this:

```
[EASYRPG]
execs = /media/data/apps/EasyRPG_gcw0_rg350.opk
romDirs = /media/sdcard/roms/EASYRPG/
romExts = .ini
```

(remember to add EASYRPG to `consoleList` at the beginning).

The problem is that it will be necessary to create covers for the EASYRPG system in the SimpleMenu themes.

**Reply by arist0brat** [reply]
The help you guys gave here plus some more google-fu on my end made it possible to get everything working! Only issue I'm having now is related to scaling-- I'm on an Anbernic RG280V, and the games don't launch in 4:3 for whatever reason-- instead, they squish up at the top of my screen. Still readable and mostly playable, but definitely wondering what might be causing it and whether or not it might be able to be fixed. I didn't see a note on the compatible games list saying it has issues displaying on smaller devices, so I'm not sure if it's just me or if it's just a known problem with the screen resolution on the rg280v.

**Reply by eduardofilo** [reply]
The only game I tried to fine tune the SimpleMenu configuration above, worked flawlessly on an RG280V, i.e. no display problems. Are you using the OPK that @a-dekker indicated in his first post?

**Reply by arist0brat** [reply]
> The only game I tried to fine tune the SimpleMenu configuration above, worked flawlessly on an RG280V, i.e. no display problems. Are you using the OPK that @a-dekker indicated in his first post?

I definitely thought I had (had it downloaded and everything) but it looks like I may have dreamt actually copying it over, lol-- that worked perfect! Thank you both for all your help.

**Reply by Shatur** [reply]
It would be awesome to have it in the image and accessible from SimpleMenu.
